### PR TITLE
Type declarations 2

### PR DESCRIPTION
**Pull request by nikku**
Attempting to improve the existing type declarations through the following:

* Using type declarations internally
* Removing private members from declared types (no need)
* Creating dedicated types even though we use default values (`CommandContext = any`)

### CODE REVIEW

File: lib/command/CommandHandler.d.ts [hunk 1]
```diff
@@ -1,10 +1,58 @@
 import { Base } from '../model';
 
+import CommandStack from './CommandStack';
+
+
+export type CommandContext = any;
```
Comment (philippfromme) [inline lib/command/CommandHandler.d.ts:6]: We're redeclaring this in multiple places.

File: test/types/test.ts [hunk 2]
```diff
@@ -358,6 +359,15 @@ eventBus.once('foo', 2000, callback);
 
 eventBus.once('foo', callback, this);
 
+const foo = diagram.invoke((modeling: Modeling, eventBus: EventBus) => {
```
Comment (philippfromme) [inline test/types/test.ts:363]: Already verified in line 145.

Comment (nikku) [reply test/types/test.ts:363]: No. This one specifically verifies the return semantics.

**Comment by nikku** [comment]
@philippfromme I guess I can close this?

**Comment by philippfromme** [comment]
Yes, this has been merged into my PR.

**Pull request closed without merging**